README.md

```diff
@@ -26,10 +26,10 @@ Main command-line arguments:
 - `(-d|--directory) <path>`: path to the temporary directory which should be
   cleaned up. Falls back to [`Path.GetTempPath`][path.get-temp-path] by default
   (which uses certain environment variables to determine the path).
-- `(-p|--period) <number>`: number of days for entry to be untouched before
+- `(-p|--period) <number>`: number of days for an entry to be untouched before
   being deleted by Vacuum. 30 by default.
-- `(-s|--space) (<number>|<number>k|<number>m|<number>g)` amount of space to clean up (`k`
-  = kibibytes, m = mebibytes, g = gibibytes). In space-cleaning mode, Vacuum will still clean
+- `(-s|--space) (<number>|<number>k|<number>m|<number>g)`: amount of space to clean up (`k`
+  = kibibytes, `m` = mebibytes, g = gibibytes). In space-cleaning mode, Vacuum will still clean
```

Comment:
Ha-ha, here you've missed your own `g` :)

(Understantably though, since you were implementing this `g` in a separate branch.)

I'll fix it and merge.